.travis.yml

```diff
@@ -21,7 +21,11 @@ install:
   - GOPATH=$PWD/Godeps/_workspace:$GOPATH go install ./...
   - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/verify-gendocs.sh
 
+before_script:
+  - npm install karma karma-junit-reporter karma-phantomjs-launcher karma-jasmine
+
 script:
+  - node_modules/karma/bin/karma start www/master/karma.conf.js --single-run --browsers PhantomJS
```

Comment:
I would prefer not to karma tests before we run hack/test-go.  Unit tests should fail first - this probably belongs either before test-cmd (because it's unit) or after test-cmd (because it's integration)